### ISSUE TITLE
Implementing on global page doesn't work

### ISSUE THREAD

**Issue by tjuarez** [RonnyWeiss/Apex-Notification-Menu-for-NavBar#19]
I'm trying to use this plugin in a huge APEX application (more than 800 pages), so I need it on the global page, but this do not work. When I create the DA in any normal page, it works perfectly, but in the global page nothing happens.
Is it possible to make the plugin to work on global page?

**Comment by walterpais81** [comment]
yes, i have it on global page. you may have something wrong in the DA, parameters or items.
